ui/components/app/collectibles-tab/collectibles-tab.test.js

```diff
@@ -137,8 +137,8 @@ const COLLECTIBLES_CONTRACTS = [
 ];
 
 const collectiblesDropdownState = {
-  0x495f947276749ce646f68ac8c248420045cb7b5e: true,
-  0xdc7382eb0bc9c352a4cba23c909bda01e0206414: true,
+  '0x495f947276749ce646f68ac8c248420045cb7b5e': true,
```

Comment:
Should this rule be applied to addresses as keys in objects? https://eslint.org/docs/latest/rules/no-loss-of-precision

Reply:
![image](https://user-images.githubusercontent.com/4448075/181080118-c308fef7-f917-48e0-a225-0476c9262a32.png)

Reply:
technically speaking, it doesn't matter, because the precision will be lost in the same way even when accessing the keys, however we don't typically pass addresses as numbers. We use strings, and this rule will help to make sure we are using strings for keys as well

Reply:
👍